src/staticanalysis/frontend/src/Tasks.vue

```diff
@@ -126,6 +126,9 @@ export default {
             <span class="tag is-info" v-else-if="task.data.state == 'cloned'">Cloned</span>
             <span class="tag is-info" v-else-if="task.data.state == 'analyzing'">Analyzing</span>
             <span class="tag is-primary" v-else-if="task.data.state == 'analyzed'">Analyzed</span>
+            <span class="tag is-danger" v-else-if="task.data.state == 'killed'">
+              Killed
```

Comment:
Nit: maybe something more descriptive, like "Killed for timeout" or "Killed because exceeded deadline"